silq/instrument_interfaces/keysight/E8267D_interface.py

```diff
@@ -381,17 +420,21 @@ def get_additional_pulses(self, interface: InstrumentInterface):
                           t_start=self.pulse.t_start - interface.envelope_padding(),
                           t_stop=self.pulse.t_stop + interface.envelope_padding(),
                           frequency=frequency_IQ,
-                          amplitude=1,
-                          phase=0,
+                          amplitude=1 + interface.I_amplitude_correction(),
+                          phase=0 + interface.I_phase_correction(),
```

Comment:
Should get rid of the 0 now.
```suggestion
                          phase=interface.I_phase_correction(),
```

Reply:
corrected